### PR TITLE
Allow multiple returns from git_checkout

### PR DESCRIPTION
**Pull request by davidhart82**

### CODE REVIEW

**Comment by davidhart82** [comment]
It doesn't look like my changes have caused the build failure?

File: thefuck/rules/git_checkout.py [hunk 1]
```diff
@@ -34,10 +34,17 @@ def get_new_command(command):
...
-    elif command.script_parts[1] == 'checkout':
-        return replace_argument(command.script, 'checkout', 'checkout -b')
+        new_commands.append(replace_argument(command.script, missing_file, closest_branch))
+    if command.script_parts[1] == 'checkout':
+        new_commands.append(replace_argument(command.script, 'checkout', 'checkout -b'))
+
+    if new_commands:
+        return new_commands
     else:
+        # if neither the better options above match, then resort to this
```
Comment (davidhart82) [inline thefuck/rules/git_checkout.py]: Now I've implemented this, I wonder whether to simply add this to the new_commands list and be done with it?

Comment (davidhart82) [reply thefuck/rules/git_checkout.py]: Maybe not, I think I've misunderstood what this last correction was doing!

**Comment by davidhart82** [comment]
I think that the match is also wrong for git_checkout; my version of git (2.24.0) doesnt have the full stop at the end.

**Comment by nvbn** [comment]
Looks good, thanks!

The ci jobs failures weren't related to the changes.

**Pull request merged**